### ISSUE TITLE
Cannot save game with keyboard only

### ISSUE THREAD

**Issue by theflyingzamboni** [Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java#563]
Because the text box overrides all alphanumeric keys for typing, it is impossible to navigate to Accept or Cancel using keyboard if d-pad is bound to WASD.

**Comment by LordMonoxide** [comment]
Tested and this is working correctly, you have to press escape to unfocus the textbox
